modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -684,13 +706,29 @@ private void _addObjectDefinitionResources(
 				).build();
 
 			for (ObjectRelationship objectRelationship : objectRelationships) {
+				String objectRelationshipName = objectRelationship.getName();
```

Comment:
@mateussgn we don't need this variable, we can just keep calling the method `objectRelationship.getName();`